htdocs/editlog.php

```diff
@@ -13,40 +13,36 @@
  *  Unicode Reminder メモ
  *****************************************************************************/
 
-require_once 'lib/consts.inc.php';
+require_once __DIR__ . 'lib/consts.inc.php';
```

Comment:
There is a / missing between `__DIR__` and ' lib/...'.

But the `__DIR__` is not needed here, and there is no `__DIR__` in all the other toplevel scripts `require`s (except for mylists.php, see discussion there).